.github/workflows/build-test.yml

```
@@ -5,24 +5,40 @@ on: [pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # Do we want to run with multiple versions of node?
+        node-version: [16]
```

@bluebill1049 It is recommended to specify which node version we use. Also, we can test RHF on multiple node versions as well. For now, I just fixed it to node 16, we can change to following in the future if it's needed
```suggestion
    strategy:
      matrix:
        # Do we want to run with multiple versions of node?
        node-version: [16, 18]
```